src/components/conversations/ConversationContainer.tsx

```diff
@@ -73,7 +73,6 @@ const ConversationContainer: React.FC<ConvoContainerProps> = (
               flexDirection: "column",
               justifyContent: "center",
               textAlign: "center",
-              fontFamily: "Inter",
```

Comment:
remove all custom overrides of font styles so they update automatically with paste.